packages/app/cypress.config.ts

```diff
@@ -11,6 +11,11 @@ export default defineConfig({
     runMode: 2,
     openMode: 0,
   },
+  // @ts-expect-error
+  clientRoute: '/cy-parent/',
```

Comment:
I have on my to do list to make it so we don't have to specify these in the parent and child configs and so that things will just work, but for now this is helping me keep track of parents, children, and potentially anywhere that we're not looking at things properly